document-store/src/main/java/org/hypertrace/core/documentstore/metric/BaseDocStoreMetricProviderImpl.java

```diff
@@ -68,9 +86,20 @@ public List<DocStoreMetric> getCustomMetrics(final CustomMetricConfig customMetr
           continue;
         }
 
+        Map<String, JsonNode> jsonNodeMap =
+            query.getSelections().stream()
+                .collect(
+                    Collectors.toUnmodifiableMap(
+                        SelectionSpec::getAlias,
+                        selectionSpec ->
+                            Optional.ofNullable(node.get(selectionSpec.getAlias()))
```

Comment:
Can you double-check if `node.get()` returns a `null` value or a `MissingNode` instance?

Reply:
Yes it gives null, checked it